### PR TITLE
Create new and move existing navigation test helper methods

### PR DESCRIPTION
**Pull request by Bhavesh-Patel**

### CODE REVIEW

File: tests/cypress/integration/reportRepair/confirmation.spec.js [hunk 1]
```diff
@@ -1,97 +1,22 @@
 import {
+  completeJourney,
   intercept_address_search,
   intercept_availability_search,
   intercept_repair_triage,
   intercept_save_repair,
```
Comment (Renny4Real) [inline tests/cypress/integration/reportRepair/confirmation.spec.js:6]: We've got inconsistent name casing. Are these all helper functions for the cypress tests alone? Perhaps we can update their name casing?

File: tests/cypress/integration/reportRepair/confirmation.spec.js [hunk 2]
```diff
@@ -1,97 +1,22 @@
...
-}
-
 describe('confirmation', () => {
   before(() => {
     intercept_availability_search();
     intercept_address_search();
     intercept_repair_triage();
     intercept_save_repair(repairID);
-    completeJourney();
+    completeJourney(false, email);
```
Comment (Renny4Real) [inline tests/cypress/integration/reportRepair/confirmation.spec.js:19]: Should we name this slightly differently? I'm thinking that `completeJourney()` and then passing `false`, `email`, doesn't immediately make sense when first reading it. 
Maybe something like `completeJourneyUsingPhone()`
Purely a suggestion though.

**Review by markWarr — APPROVED** [review]

**Review by Renny4Real — APPROVED** [review]

File: tests/cypress/integration/reportRepair/confirmation.spec.js (hunk 2 above)
Comment (agnieszkazbikowska-madetech) [reply tests/cypress/integration/reportRepair/confirmation.spec.js:19]: I agree with Renny's suggestion, it seems more meaningful.

Comment (Bhavesh-Patel) [reply tests/cypress/integration/reportRepair/confirmation.spec.js]: Good suggestion!

Have implemented now.

File: tests/cypress/integration/reportRepair/confirmation.spec.js (hunk 1 above)
Comment (Bhavesh-Patel) [reply tests/cypress/integration/reportRepair/confirmation.spec.js]: Yes, these are all Cypress test helpers.

To keep the changes in this PR focused, making names consistent will be part of another PR.

**Review by markWarr — APPROVED** [review]

**Comment by Bhavesh-Patel** [comment]
[Do you pay rent to the council? (Common)](https://trello.com/c/gRIF2ili/147-do-you-pay-rent-to-the-council-common)

**Pull request merged**